modules/lazyload/assets/js/frontend.js

```diff
@@ -0,0 +1,20 @@
+document.addEventListener( 'DOMContentLoaded', function() {
+    const dataAttribute = 'data-e-bg-lazyload';
+    const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
+    const lazyloadBackgroundObserver = new IntersectionObserver( ( entries ) => {
+        entries.forEach( ( entry ) => {
+            if ( entry.isIntersecting ) {
+                let [ lazyloadBackground, element ] = [ entry.target, entry.target ];
+                const lazyloadSelector = element.querySelector( '.elementor-widget-container' );
```

Comment:
Use a data-attribute

Reply:
Be prepared for nested widgets

Reply:
*The Document method querySelector() returns the first Element within the document that matches the specified selector*

Reply:
Maybe the parent doesn't have a background, but the nested has